attest/ake/src/lib.rs

```diff
@@ -75,6 +75,9 @@ mod test {
             .verify_quote(&quote, None)
             .expect("Could not sign our bogus report");
 
+        // TODO: replace with dcap
+        let attestation_evidence = EvidenceKind::Epid(attestation_evidence);
```

Comment:
I think there should be another test that tests the DCAP handshake.

Reply:
In a PR that I'm going to start rebasing here, has a DCAP test, but it removes the IAS test.

@eranrund, do you think we should maintain both?

Reply:
I think so, at least until we retire EPID.

Reply:
The test will be coming in when the enclaves are generating dcap evidence